src/plugins/expressions/common/expression_functions/types.ts

```diff
@@ -94,3 +95,30 @@ export interface ExpressionFunctionDefinition<
  * Type to capture every possible expression function definition.
  */
 export type AnyExpressionFunctionDefinition = ExpressionFunctionDefinition<any, any, any, any>;
+
+/**
+ * Anybody registering an expression function can add a property to this
+ * interface so that any module importing ExpressionFunctionDefinitions can
+ * have the interfaces of all expression functions available to them.
+ *
+ *  declare module '../../plugins/expressions/public' {
+ *    interface ExpressionFunctionDefinitions {
+ *      myExprFunctionName?: MyExpressionFunctionDefinition;
+ *    }
+ *  }
+ *
+ *  declare module '../../plugins/expressions/server' {
+ *    interface ExpressionFunctionDefinitions {
+ *      myExprFunctionName?: MyExpressionFunctionDefinition;
+ *    }
+ *  }
+ *
+ * @public
+ */
+export interface ExpressionFunctionDefinitions {
+  // The property name should match the name of your function.
+  kibana_context: ExpressionFunctionKibanaContext;
+
```

Comment:
All of the other functions provided by the expressions plugin would need to be added here too